### ISSUE TITLE
Test llvm_tools::tests::test_profraws_to_lcov fails on MacOS 11.6

### ISSUE THREAD

**Issue by brendandahl** [mozilla/grcov#713]
It appears `llvm-profdata` doesn't work with the test file in the repo:

```
warning: test/default.profraw: unsupported instrumentation profile format version\nerror: no profile can be merged
```

**Comment by marco-c** [comment]
We likely need to replace it with a test file in the newer format (I guess LLVM was recently updated in Rust).

**Comment by marco-c** [comment]
Fixed by 9ad23c1f4ec98401f6db9c7238b184bea9aedfe9.
